zerver/openapi/zulip.yaml

```diff
@@ -12821,6 +12836,24 @@ paths:
                           - 2 = Nobody
 
                           **Changes**: New in Zulip 3.0 (feature level 1).
+                      realm_direct_message_initiator_group_id:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          The ID of the user group in the organization which has the permission
+                          to initiate direct message threads.
```

Comment:
We should spell this out more precisely. Maybe just add a second sentence: "This policy prevents users who are not in this group from sending direct messages to a direct message recipient (individual or group) with which they have not previously exchanged messages")